### PR TITLE
APP-3131: Ruby 2.3.8

### PR DESCRIPTION
**Pull request by Gubriels**
Ticket: https://navigatingcancer.atlassian.net/browse/APP-3131

### CODE REVIEW

**Comment by Gubriels** [comment]
Passing tests:
Screen Shot 2020-05-11 at 10 54 20 AM

File: attribute_normalizer.gemspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.name = %q{attribute_normalizer}
   s.version =AttributeNormalizer::VERSION
 
-  s.required_rubygems_version = Gem::Requirement.new("> 1.3.1") if s.respond_to? :required_rubygems_version=
+  s.required_rubygems_version = Gem::Requirement.new(">= 2.3.8") if s.respond_to? :required_rubygems_version=
```
Comment (Gubriels) [inline attribute_normalizer.gemspec:9]: This deviates slightly from what the ticket specifies(`s.required_ruby_version = '>= 2.3.8'`) but I opted for minimal changes and left their implementation.

**Comment by Gubriels** [comment]
Passing specs:
Screen Shot 2020-05-11 at 10 57 47 AM

File: attribute_normalizer.gemspec (hunk 1 above)
Comment (ninjascribble) [reply attribute_normalizer.gemspec:9]: Weird, yeah that's fine though

**Review by ninjascribble — APPROVED** [review]
Looks good!

**Pull request merged**